chess-AI/options.py

```diff
@@ -28,3 +28,26 @@ def __init__(self, learning_rate, momentum, weight_decay, epochs, batch_size, ga
         self.save_path = save_path
         self.num_saved_models = num_saved_models
         self.overwrite = overwrite
+
+
+class MCTSOptions(TrainOptions):
+
```

Comment:
Pls add class docstring, only need to describe attributes that are not in `TrainOptions` but pls mention that most of the attributes are described in the `__init__` of `TrainOptions`.